circle.yml

```diff
@@ -0,0 +1,16 @@
+dependencies:
+  pre:
+    - echo y | android update sdk --no-ui -a --filter tools
+    - echo y | android update sdk --no-ui -a --filter platform-tools
+    - echo y | android update sdk --no-ui -a --filter android-24
+    - echo y | android update sdk --no-ui -a --filter build-tools-24.0.2
+    - echo y | android update sdk --no-ui -a --filter extra-google-google_play_services
+    - echo y | android update sdk --no-ui -a --filter extra-google-m2repository
+    - echo y | android update sdk --no-ui -a --filter extra-android-m2repository
+    - echo y | android update sdk --no-ui -a --filter build-tools-24.0.1
+    - echo y | android update sdk --no-ui -a --filter build-tools-24.0.0
```

Comment:
Doesn't CircleCI detect all this automatically? Also, former versions of build tools aren't needed.